### PR TITLE
Fix search input persistence after submission

### LINKED ISSUE #331: Search query doesn't persist in search bar
**Describe the bug**
When searching and pressing enter, the search is submitted but the original search text disappears from the search bar.

### PR DESCRIPTION
**Pull request by abetco**
## Summary

Fixes a bug where the search text was not saved within the search input after submitting search
Closes #331 

## Changes

- On the search projects page, now displays the last query in the search bar

## Screenshots

Before:
image

After:
image

### CODE REVIEW

**Comment by NdibeRaymond** [comment]
this looks great @abetco, We will merge this after the private_mode PR has been merged and all resulting conflicts fixed.

**Review by ghost — APPROVED** [review]
Incredible work Albert! I think I'm going to assign you to some more issues :). Feel free to squash and merge.

**Pull request merged**
